index.html

```diff
@@ -1,12 +1,76 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на интенсивном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">Базовый HTML и CSS</a>».</p>
-
-  </body>
+<html lang="ru" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Девайс</title>
+</head>
+<body>
+<header class="header">
+  <div class="header1">
+    <ul>
+      <li><button class="search">poisk po saitu</button></li>
+      <li><button class="enter">voiti</button></li>
+      <li><button class="compare">sravnit</button></li>
+      <li><button class="basket">basket</button></li>
+    </ul>
+  </div>
+  <div class="header2">
+    <ul><!--задать один стиль всем ссылкам, где ставить class-->
+      <li> <a href="">catalog</a></li>
+      <li> <a href="">dostavka</a></li>
+      <li> <a href="">garantiya</a></li>
+      <li> <a href="">kontakti</a></li>
+    </ul>
+  </div>
+</header>
+<div class="gallery">
+  <!-- не знаю как сделать галерею с слайдами,
```

Comment:
We need to add at least one element of gallery here at least.
You can do it like this:
```
<ul>
  <li>This is promo action XXX</li>
</ul>
```